### PR TITLE
Fix/profile

### PR DESCRIPTION
**Pull request by YevheniiBieliaiev**

### CODE REVIEW

File: packages/frontend/components/profile/user-account/personal-info/edit-form.tsx [hunk 1]
```diff
@@ -92,11 +99,12 @@ const EditPersonalInfo = ({ user }: { user: FullUserProfileDto }) => {
               size="flexible"
               type="button"
               variant="outlined"
+              disabled={loading}
               onClick={onResetHandler}
             >
               {t('personal-info:action.cancel')}
             </Button>
-            <Button size="flexible" type="submit">
+            <Button size="flexible" type="submit" disabled={loading}>
```
Comment (Nightlngale) [inline packages/frontend/components/profile/user-account/personal-info/edit-form.tsx:107]: add loader while is loading

**Review by Nightlngale — CHANGES_REQUESTED** [review]

File: packages/frontend/components/auth/sign-up/component.tsx [hunk 2]
```diff
@@ -18,15 +19,35 @@ export const SignUpForm = () => {
...
   };
 
+  const onChangeHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const value = event.target.value;
+    clearErrors('password');
+    if (value.includes(' ')) {
+      setError('password', {
+        message: t(UserPersonalInfoValidationMessage.SPACES_IN_PASSWORD),
+      });
+    } else if (/^[А-ЯЁIЇҐЄЂЃЀЅЍЈЉЊЋЌЎа-яёіїґєђѓѐѕѝјљњћќў]+$/.test(value)) {
```
Comment (Nightlngale) [inline packages/frontend/components/auth/sign-up/component.tsx:42]: add file "utils.ts" inside current directory and move it there

File: packages/frontend/components/profile/user-account/personal-info/edit-form.tsx [hunk 3]
```diff
@@ -51,20 +60,18 @@ const EditPersonalInfo = ({ user }: { user: FullUserProfileDto }) => {
       setError('newPassword', {
         message: t(UserPersonalInfoValidationMessage.CYRILLIC),
       });
-    } else {
+    } else if (
+      !/^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])[\da-zA-Z~!@#$%^*\-_=+[{\]}/;:,.?]+$/.test(
```
Comment (Nightlngale) [inline packages/frontend/components/profile/user-account/personal-info/edit-form.tsx:64]: same as above

**Review by Nightlngale — CHANGES_REQUESTED** [review]

File: packages/frontend/components/profile/user-account/personal-info/edit-form.tsx (hunk 1 above)
Comment (YevheniiBieliaiev) [reply packages/frontend/components/profile/user-account/personal-info/edit-form.tsx:107]: ok

**Review by Nightlngale — APPROVED** [review]

**Pull request merged**